deploy/example/e2e_usage.md

```diff
@@ -6,9 +6,9 @@
 kubectl create -f https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/master/deploy/example/storageclass-azuredisk-csi.yaml
 ```
 
- - Create an azuredisk CSI PVC
+ - Create a statefulset with Azure Disk mount
 ```
-kubectl create -f https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/master/deploy/example/pvc-azuredisk-csi.yaml
+kubectl create -f https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/master/deploy/example/statefulset.yaml
```

Comment:
If using statefulset, it seems that the other two parts need to be rewrote.

Reply:
which part? For static provisioning, we still use `pvc-azuredisk-csi.yaml`. And for dynamic provisioning, use statefulset example is easier to demo.

Reply:
let's merge this PR first, will modify the content in another PR.

Reply:
2. validate PVC status and create an nginx pod
3. enter the pod container to do validation